daemon/cmd/kube_proxy_healthz_test.go

```diff
@@ -17,7 +17,7 @@ import (
 	jsoniter "github.com/json-iterator/go"
 )
 
-var json = jsoniter.ConfigCompatibleWithStandardLibrary
+var json = jsoniter.ConfigFastest
```

Comment:
Something to note about this commit is the number of allocs/op. They actually increase in the fastest config, but I didn't dig too deep into that. I'm not sure of the impact this may have in memory-sensitive environments, but the overall usage of memory is reduced. I figured that might be enough of a win + the CPU improvement was worthy enough to submit this even though I don't quite understand the allocs/op just yet.